bark/benchmark/benchmark_runner.py

```diff
@@ -289,3 +305,13 @@ def _log_eval_average(self, result_dct_list, configs):
         self.logger.info("\n------------------- Current Evaluation Results ---------------------- \n Num. Results:{}\n {} \n \
 ---------------------------------------------------------------------".format(len(result_dct_list),
                                                                               grouped.to_string()))
+
+    def _get_track_id_and_filename(self, scenario):
+        # only relevant for scenarios from dataset
```

Comment:
This should not be in benchmark runner (is not using self argument). Thus should be some free function somewhere in the interaction dataset scenario generation.

Reply:
I don't think it makes sense to move it there, as I would not have control over it after being processed to the database. Essentially, the track file will change, as a benchmark will use multiple ones (that's why I want to log which one is used)

Reply:
I moved it to the scenario class